### PR TITLE
Build everything locally without Docker

### PR DESCRIPTION
**Pull request by philix**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,35 +1,55 @@
-EMCC = docker run --rm -v $(shell pwd):/src emcc_sdl emcc
-EMCCL = docker run --rm -v $(shell pwd)/node_modules/lua-src/src:/src emcc_sdl emcc
-FLAGS = node_modules/lua-src/src/liblua.a \
+CC=./node_modules/emcc/emsdk/emscripten/1.38.27/emcc
+AR=./node_modules/emcc/emsdk/emscripten/1.38.27/emar
+RANLIB=./node_modules/emcc/emsdk/emscripten/1.38.27/emranlib
+CONFIGURE=./node_modules/emcc/emsdk/emscripten/1.38.27/emconfigure
+MAKE=./node_modules/emcc/emsdk/emscripten/1.38.27/emmake
+
+CFLAGS =
```
Comment (philix) [inline Makefile]:
```suggestion
CFLAGS =\
```

**Comment by hensmith** [comment]
okay so I changed these two bits at the bottom of the makefile and it worked for me

```Makefile

itsy.js: lua
	$(CC) node_modules/b64/decode.c -o b64_decode.o
	$(CC) node_modules/b64/encode.c -o b64_encode.o
	$(CC) -Inode_modules/lua-src/src -I node_modules/b64 itsy.c $(OBJS) $(EMCCFLAGS) -o itsy.js

lua:
	#$(EMMAKE) make -C node_modules/lua-src/src generic
	cd node_modules/lua-src/src && make generic CC='../../../$(CC)'
```

gonna merge this then apply those changes in master 🚀🚀🚀🚀

**Comment by hensmith** [comment]
(I mean I am going to figure out which of these arguments that I frantically added until it worked actually helped and tidy this up btw not just immediately take a shit on this nice clean new Makefile 😆)

**Comment by hensmith** [comment]
okay it was just the ordering of the arguments! `$(CC) $(CFLAGS) itsy.c $(OBJS) $(EMCCFLAGS) -o itsy.js` works and `$(CC) $(CFLAGS) $(EMCCFLAGS) $(OBJS) itsy.c -o itsy.js` doesn't!

**Pull request merged**